cmds/task/actors.go

```diff
@@ -13,34 +13,92 @@ import (
 )
 
 // runCancel cancels the runs of a given task.
-func runCancel(credentials *tcclient.Credentials, args []string, out io.Writer, _ *pflag.FlagSet) error {
+func runCancel(credentials *tcclient.Credentials, args []string, out io.Writer, flagSet *pflag.FlagSet) error {
+	noop, _ := flagSet.GetBool("noop")
+	confirm, _ := flagSet.GetBool("confirm")
+
 	q := makeQueue(credentials)
 	taskID := args[0]
 
+
+	if noop {
+		displayNoopMsg("Would cancel", credentials, args)
+		return nil
+	}
+
+	if confirm {
+		var response = confirmMsg("Cancels", credentials, args)
+		if response == "y" {
+			c, err := q.CancelTask(taskID)
+			run := c.Status.Runs[len(c.Status.Runs)-1]
+			if err != nil {
```

Comment:
this err check should come before trying to use `c` (which is probably nil in the error case, meaning the previous line will panic)